pom.xml

```diff
@@ -48,6 +49,9 @@
     <!-- osgi -->
     <osgi.export.packages>org.fcrepo.client.*;version=${project.version}</osgi.export.packages>
     <osgi.import.packages>*</osgi.import.packages>
+    <grizzly.version>2.3.18</grizzly.version>
+    <spring.version>4.1.4.RELEASE</spring.version>
+    <logback.version>1.1.2</logback.version>
```

Comment:
very minor point here, but can you move these dependencies higher up in the list, so as to keep them separate from the OSGi definitions.